spiped/dispatch.c

```diff
@@ -95,13 +105,31 @@ doaccept(struct accept_state * A)
 
 /* A connection has closed.  Accept more if necessary. */
 static int
-callback_conndied(void * cookie)
+callback_conndied(void * cookie_node)
```

Comment:
We call it the principle of least astonishment (POLA).  But to address the question, that's why the first local variable declaration in cookie-using functions should be `struct foo * bar = cookie;`.

... I guess that could do with documenting in STYLE, actually.